app/pendrive_detection/pendrive_detector.py

```diff
@@ -0,0 +1,125 @@
+import os
+import sys
+from typing import Optional
+import psutil
```

Comment:
as I see you're using psutil.only in linux function so it can be moved to the if statement below